### ISSUE TITLE
SwipeCardView not looping items 

### ISSUE THREAD

**Issue by imranemi1221** [Arjun-sna/android-swipecards-view#2]
In SwipeCardView when to swipe then removes the item, Is it possible to loop back the items? So that the adapter will never get empty.

**Comment by phani51** [comment]
Yes, u can do that by adding the item to the same list when swiping the card.
